src/matrix_t/matrix_t.h

```diff
@@ -26,9 +27,10 @@ class matrix_t {
     channel = get_channel(data_type) | 0;
     _data_type_size.assign(
         {-1, 1, 4, -1, 4, -1, -1, -1, 8, -1, -1, -1, -1, -1, -1, -1, 8});
-    size = (cols * get_data_type_size(data_type) * channel) *
-           rows; //(cols * rows);
-    dt = new data_t((cols * get_data_type_size(type) * channel) * rows);
```

Comment:
As said in the previous comment that is not correct (also for the data_t class).